wordpress-mobile/WordPress-iOS
# Fix UI Tests for iOS 13/Xcode 11

This PR has updates to ensure stability for local UI test runs:

* Fixes the selector for the Safari URL field (used in the magic link login and signup tests).
* Adds a build phase that disables hardware keyboards, so any tests using keyboard presses on the software keyboard will work as expected. (This also closes any open simulators to apply that change.)

It also relies on WordPressMocks updates in this PR:

* https://github.com/wordpress-mobile/WordPressMocks/pull/10

To test:

* Confirm the UI tests pass on CircleCI.
* Run the UI tests locally and confirm they pass (even if you previously disabled the software keyboard/connected a hardware keyboard in the simulator).

Update release notes:
